cmd/flow-dps-indexer/README.md

```diff
@@ -12,6 +12,7 @@ The index is generated in the form of a Badger database that allows random acces
 Usage of flow-dps-indexer:
   -c, --checkpoint string   checkpoint file for state trie
   -d, --data string         database directory for protocol data
+  -f, --force bool          overwrite existing index database
```

Comment:
Missing the default value?

Reply:
Since it's the default value of a bool to be false (i think that's why at least), it's not shown in the output:

```
./flow-dps-indexer --help
Usage of ./flow-dps-indexer:
  -c, --checkpoint string   checkpoint file for state trie
  -d, --data string         database directory for protocol data
  -f, --force               overwrite existing index database
  -i, --index string        database directory for state index (default "index")
  -l, --level string        log output level (default "info")
  -t, --trie string         data directory for state ledger
```

I'll add it manually but we'll have to be careful not to overwrite.